cmake travis error

Error in.travis when cmake for google benchmark has been started

```
CMake 2.8.11 or higher is required. You are running version 2.8.7
```

@JaroslawWiosna

I use a hint from [https://github.com/travis-ci/travis-ci/issues/4631#issuecomment-191153634](https://github.com/travis-ci/travis-ci/issues/4631#issuecomment-191153634)
Log from [MatrixLibs/builds/171604478](https://travis-ci.org/JaroslawWiosna/MatrixLibs/builds/171604478)

> make
> Scanning dependencies of target benchmark
> [  3%] Building CXX object src/CMakeFiles/benchmark.dir/complexity.cc.o
> [  7%] Building CXX object src/CMakeFiles/benchmark.dir/timers.cc.o
> [ 10%] Building CXX object src/CMakeFiles/benchmark.dir/commandlineflags.cc.o
> [ 14%] Building CXX object src/CMakeFiles/benchmark.dir/csv_reporter.cc.o
> [ 17%] Building CXX object src/CMakeFiles/benchmark.dir/sysinfo.cc.o
> [ 21%] Building CXX object src/CMakeFiles/benchmark.dir/colorprint.cc.o
> [ 25%] Building CXX object src/CMakeFiles/benchmark.dir/reporter.cc.o
> [ 28%] Building CXX object src/CMakeFiles/benchmark.dir/string_util.cc.o
> [ 32%] Building CXX object src/CMakeFiles/benchmark.dir/sleep.cc.o
> [ 35%] Building CXX object src/CMakeFiles/benchmark.dir/json_reporter.cc.o
> [ 39%] Building CXX object src/CMakeFiles/benchmark.dir/console_reporter.cc.o
> [ 42%] Building CXX object src/CMakeFiles/benchmark.dir/benchmark_register.cc.o
> [ 46%] Building CXX object src/CMakeFiles/benchmark.dir/benchmark.cc.o
> Linking CXX static library libbenchmark.a
> [ 46%] Built target benchmark
> Scanning dependencies of target basic_test
> [ 50%] Building CXX object test/CMakeFiles/basic_test.dir/basic_test.cc.o
> Linking CXX executable basic_test
> ../src/libbenchmark.a(timers.cc.o): In function `benchmark::ProcessCPUUsage()':
> timers.cc:(.text+0x105): undefined reference to`clock_gettime'
> ../src/libbenchmark.a(timers.cc.o): In function `benchmark::ThreadCPUUsage()':
> timers.cc:(.text+0x14f): undefined reference to`clock_gettime'
> collect2: error: ld returned 1 exit status
> make[2]: **\* [test/basic_test] Error 1
> make[1]: **\* [test/CMakeFiles/basic_test.dir/all] Error 2
> make: **\* [all] Error 2
> The command "make" failed and exited with 2 during .
> Your build has been stopped.

@michal915 It seems that this is no longer cmake error, but benchmark error. I am assigning this task to you then :D

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,7 +11,7 @@ addons:
     apt:
         sources:
             - ubuntu-toolchain-r-test
-            i smspillaz/cmake-2.8.12
+            - smspillaz/cmake-2.8.12
         packages:
             - gcc-5
             - g++-5

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,6 +11,7 @@ addons:
     apt:
         sources:
             - ubuntu-toolchain-r-test
+            i smspillaz/cmake-2.8.12
         packages:
             - gcc-5
             - g++-5

--- a/.travis.yml
+++ b/.travis.yml
@@ -16,7 +16,7 @@ addons:
             - gcc-5
             - g++-5
             - libgtest-dev
-            - cmake=2.8.11
+            - cmake
 
         coverity_scan:
             name: MatrixLibs

--- a/.travis.yml
+++ b/.travis.yml
@@ -15,7 +15,7 @@ addons:
             - gcc-5
             - g++-5
             - libgtest-dev
-            - cmake "2.8.11"
+            - cmake=2.8.11
 
         coverity_scan:
             name: MatrixLibs

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,7 +11,7 @@ addons:
     apt:
         sources:
             - ubuntu-toolchain-r-test
-            - smspillaz/cmake-2.8.12
+            - smspillaz
         packages:
             - gcc-5
             - g++-5

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,12 +11,13 @@ addons:
     apt:
         sources:
             - ubuntu-toolchain-r-test
-            - smspillaz
+            - george-edison55-precise-backports
         packages:
             - gcc-5
             - g++-5
             - libgtest-dev
             - cmake
+            - cmake-data
 
         coverity_scan:
             name: MatrixLibs

--- a/.travis.yml
+++ b/.travis.yml
@@ -15,6 +15,7 @@ addons:
             - gcc-5
             - g++-5
             - libgtest-dev
+            - cmake
 
         coverity_scan:
             name: MatrixLibs
@@ -31,6 +32,7 @@ before_script:
     - cmake ..
     - make
     - sudo make install
+    - cd ../..
     - "cd /usr/src/gtest && sudo cmake . && sudo cmake --build . && sudo mv libg* /usr/local/lib/ ; cd -"
 
 script: 

--- a/.travis.yml
+++ b/.travis.yml
@@ -15,7 +15,7 @@ addons:
             - gcc-5
             - g++-5
             - libgtest-dev
-            - cmake
+            - cmake "2.8.11"
 
         coverity_scan:
             name: MatrixLibs

